### PR TITLE
Update [Recovery Method] in translations

### PR DESCRIPTION
**Pull request by phavekes**

### CODE REVIEW

File: translations/messages.nl_NL.xliff [hunk 1]
```diff
@@ -206,7 +206,7 @@
       </trans-unit>
       <trans-unit id="4c78999da0bb06ba34dc4110f9237678c54e28fc" resname="ra.auditlog.recovery_token_type">
         <source>ra.auditlog.recovery_token_type</source>
-        <target state="translated">[Recovery token] type</target>
+        <target state="translated">Herstelmethode type</target>
```
Comment (thijskh) [inline translations/messages.nl_NL.xliff:209]: `Type herstelmethode`

File: translations/messages.nl_NL.xliff [hunk 2]
```diff
@@ -1231,7 +1231,7 @@
       </trans-unit>
       <trans-unit id="7cd6e960fe366ea7f42a30e2b9f37e8a51a971b7" resname="ra.recovery_token.search.text.no_recovery_tokens">
         <source>ra.recovery_token.search.text.no_recovery_tokens</source>
-        <target state="translated">Er zijn geen [recovery tokens] gevonden</target>
+        <target state="translated">Er zijn geen herstelmethoden gevonden</target>
```
Comment (thijskh) [inline translations/messages.nl_NL.xliff:1234]: `herstelmethodes`

File: translations/messages.nl_NL.xliff [hunk 3]
```diff
@@ -1241,7 +1241,7 @@
       </trans-unit>
       <trans-unit id="c6a8ca63fb65eca2a27a29e12c249f2426bf9bfa" resname="ra.recovery_token.search.title">
         <source>ra.recovery_token.search.title</source>
-        <target state="needs-l10n">[Recovery tokens]</target>
+        <target state="needs-l10n">herstelmethoden</target>
```
Comment (thijskh) [inline translations/messages.nl_NL.xliff:1244]: `Herstelmethodes` (met hoofdletter ook?)

File: translations/messages.en_GB.xliff [hunk 4]
```diff
@@ -206,7 +206,7 @@
       </trans-unit>
       <trans-unit id="4c78999da0bb06ba34dc4110f9237678c54e28fc" resname="ra.auditlog.recovery_token_type">
         <source>ra.auditlog.recovery_token_type</source>
-        <target state="translated">[Recovery Token] type</target>
+        <target state="translated">Recoverymethod type</target>
```
Comment (thijskh) [inline translations/messages.en_GB.xliff:209]: `Recovery method`

**Review by MKodde — COMMENTED** [review]
Translations are looking great. Thumbs up, and approval there.

While grepping for translation to-do left-overs I ran into this untranslated occurrence:

https://github.com/OpenConext/Stepup-RA/blob/a500270eb8f639f2cac04a44247be59bff7146f8/translations/messages.en_GB.xliff#L1617

Intentional?

**Pull request merged**